de.dlr.sc.virsat.model.extension.cef/src/de/dlr/sc/virsat/model/extension/cef/model/EquipmentParameters.java

```diff
@@ -61,11 +61,12 @@ public EquipmentParameters(CategoryAssignment categoryAssignment) {
 	 */
 	public long getGlobalUnitQuantity() {
 		long globalUnitQuantity = 1;
-		try {
-			globalUnitQuantity *= getUnitQuantity();
-		} catch (NumberFormatException e) {
-			globalUnitQuantity *= 1;
-		}
+		
+		// Workaround due to API Inconsistency with Beans see https://github.com/virtualsatellite/VirtualSatellite4-CEF/issues/99
+		Long unitQauntity = getUnitQuantityBean().getValue();

```

Comment:
Let's maybe do it something like
`globalUnitQuantity *= isSetQuantity() ? getQuantity() : 1;`
Then it is not a temporary workaround anymore :)